client/src/components/common/AppHeader/AppHeader.tsx

```diff
@@ -12,16 +12,17 @@ const AppHeader = ({history}: any) => {
 
     return (
         <header className={"app-header"}>
+            <NavLink to={"/main/home"} >
             <div className={"app-branding"}>
-                <img src={logo} alt={''}/>
+                    <img  src={logo} alt={''}/>
             </div>
+            </NavLink>
             <div className={"app-options"}>
                 <div style={{display: 'flex', flexDirection: 'row', height: '100%'}}>
-                    <NavLink to={"/main/home"} className={"app-option"}>Home</NavLink>
-                    <NavLink to={"/main/profile"} className={"app-option"}>Profile</NavLink>
                 </div>
-                <div className={'logout'} onClick={() => handleLogout()}>Logout</div>
             </div>
+            <NavLink to={"/main/profile"} className={"profile"}>Profile</NavLink>
+            <div className={'logout'} onClick={() => handleLogout()}>Logout</div>
         </header>
```

Comment:
Formatting! It is specially relevant when writing HTML (lots of nested components), as it might lead to misunderstandings when reading it. Please remember to run a code reformat on changed files. To do this you can run Code > Reformat Code on IntelliJ (CMD + Shift + L)

Reply:
Also, please add on hover style to Profile, same as Logout!